--- make.inc.example	2025-07-14 19:24:33.424071628 +0000
+++ make.inc	2025-07-18 17:49:13.619575983 +0000
@@ -56,16 +56,9 @@

 else

-    # LAPACK
-    LINKS        += $(HOME)/opt/lapack/liblapack.a
-
-    # BLAS
-
-    # Ref BLAS
-    #LINKS        += ${HOME}/opt/lapack/libblas.a
-
     # OpenBLAS
-    LINKS        += ${HOME}/opt/openblas/lib/libopenblas.a
+    LINKS        += /lib64/liblapack.so.3
+    LINKS        += /lib64/libblas.so.3
+
 endif